examples/docker-compose.yml.chat_example

```diff
@@ -16,17 +16,17 @@ services:
       # Media server url:
       - BBB_STREAM_URL=rtmp://media_server_url/stream/stream_key
       # Enable chat functionality
-      -BBB_ENABLE_CHAT=true
+      - BBB_ENABLE_CHAT=true
       # show chat in live stream
-      -BBB_SHOW_CHAT=false
+      - BBB_SHOW_CHAT=false
       # Set REDIS host (default: 'redis')
-      -BBB_REDIS_HOST=redis
+      - BBB_REDIS_HOST=redis
       # Set REDIS channel to subscribe (default: 'chat')
-      -BBB_REDIS_CHANNEL=chat
+      - BBB_REDIS_CHANNEL=chat
       # Username for the chat (default: 'Chat')
```

Comment:
thanks for the fix - one more line that misses the space. cheers

Reply:
thanks, changed it!